CMakeLists.txt

```diff
@@ -82,6 +82,14 @@ add_halide_library(backprojection_cuda FROM backprojection.generator
                                               blocksize=16
                                               blocksize_gpu_tile=16
                                               blocksize_gpu_split_x=16)
+add_halide_library(backprojection_cuda_split FROM backprojection.generator
```

Comment:
pedantic: whitespace-align the FEATURES and PARAMS options with FROM, like you did below.

Reply:
Done.